api/routes/api/restaurants.route.js

```diff
@@ -9,7 +9,7 @@ router
   .get(restaurantController.getRestaurants);
 
 router
-  .route('/:restaurantId')
+  .route('/:APIID')
```

Comment:
Needs to stay as restaurantId. Changing this to APIID will break the patch and delete routes.